lib/releases.js

```diff
@@ -127,69 +163,25 @@ module.exports.generateReleaseInfo = ({
   mergedPullRequests
 }) => {
   let body = config.template
-  const [categoriesConfig, orderedCategories] = getCategoriesConfig({ config })
-
-  if (!lastRelease) {
-    body = body.replace('$PREVIOUS_TAG', '')
-  } else {
-    body = body.replace('$PREVIOUS_TAG', lastRelease.tag_name)
-  }
-
-  if (mergedPullRequests.length === 0) {
-    body = body.replace('$CHANGES', config['no-changes-template'])
-  } else {
-    const categorizedPullRequests = categorizePullRequests(
-      mergedPullRequests,
-      orderedCategories
-    )
 
-    body = body.replace(
-      '$CHANGES',
-      orderedCategories
-        .reduce((acc, category, index) => {
-          if (
-            !categorizedPullRequests[category] ||
-            categorizedPullRequests[category].length === 0
-          )
-            return acc
-
-          if (category !== UNCATEGORIZED) {
-            acc.push(`## ${categoriesConfig[category].title}\n\n`)
-          }
-
-          acc.push(
-            categorizedPullRequests[category]
-              .map(pullRequest =>
-                config['change-template']
-                  .replace('$TITLE', pullRequest.title)
-                  .replace('$NUMBER', pullRequest.number)
-                  .replace('$AUTHOR', pullRequest.user.login)
-              )
-              .join('\n')
-          )
-
-          if (index + 1 !== orderedCategories.length) acc.push('\n\n')
-
-          return acc
-        }, [])
-        .join('')
-    )
-  }
-
-  body = body.replace(
-    '$CONTRIBUTORS',
-    contributorsSentence({ commits, pullRequests: mergedPullRequests })
-  )
+  body = template(body, {
+    $PREVIOUS_TAG: _.get(lastRelease, 'tag_name') || '',
```

Comment:
Is this the only place we use `lodash` now? Why don't we do something like this, and remove the dependency (one less thing to update, have security bugs, etc):

```suggestion
    $PREVIOUS_TAG: lastRelease ? lastRelease.tag_name : ''
```

Reply:
Well lodash is being suggested in #164 but I'll happily make the change :smile:

Reply:
Ah, I see! Thought I saw that somewhere :)